lib/modules/datasource/npm/get.ts

```diff
@@ -193,6 +197,9 @@ export async function getDependency(
       ) {
         release.sourceDirectory = source.sourceDirectory;
       }
+      if (dep.deprecationMessage) {
+        release.isDeprecated = true;
+      }
```

Comment:
This is the actual fix, but it wouldn't work properly without the cache busting above